### ISSUE TITLE
[Question]: Enabled/disabled debugging mode

### ISSUE THREAD

**Issue by Kiruel** [nvim-flutter/flutter-tools.nvim#414]
Hello,

Quick question here: Is it possible to enable/disable the dap without changing this variable every time we need to use the debugger?

```
debugger = {
        enabled = true, // This one ?
},
```

I cannot find any solution to run the project in debug mode without changing this every time and re-opening nvim.

Thanks!

**Comment by sidlatau** [comment]
No, there is currently no option to change the `debugger.enabled` value. However, I am considering adding a `:FlutterDebug` command for this purpose. This command would not depend on the `debugger.enabled` setting and could always start Flutter in debug mode. Would this approach work for you?

**Comment by sidlatau** [comment]
I reopened this case to ensure it is not forgotten :)

**Comment by sidlatau** [comment]
By the way, is there any particular reason you're switching between debug and non-debug modes, @Kiruel ? I recently fixed some commands that weren't working in debug mode (e.g., "Inspect widget"). Is there anything else that needs fixing?

**Comment by Kiruel** [comment]
Yes during dev, on a big codebase I switch between this two. It's common also to get some information without print everywhere.
For now I don't really use debugger in neovim because I need to change the variable reopen open.
But using vscode instead when I use the debugger.
Have not try during unit test also but will test if you need me on that.

**Comment by Kiruel** [comment]
I will mention also if you add the debugger via flutterdebug it should be possible to hotreaload right ?

**Comment by sidlatau** [comment]
Yes, hot reload/hot restart works in debug mode. I use debug mode as my default mode, it works without problems.

**Comment by sidlatau** [comment]
Hi, @kamskry, I made `Open Dev Tools` command work in debug mode via:
https://github.com/nvim-flutter/flutter-tools.nvim/pull/419
Could you check if it works for you?

**Comment by kamskr** [comment]
@sidlatau Yes, it works! Thank you so much, this is amazing

**Comment by Kiruel** [comment]
Do I need to wait a new release to test it @sidlatau or I can switch to use main ?

**Comment by sidlatau** [comment]
Please use the `main` branch for now. New releases are on hold while we wait for https://github.com/nvim-flutter/flutter-tools.nvim/issues/393 to be resolved.
